vue/components/organisms/pickers/pickers.vue

```diff
@@ -665,6 +701,39 @@ export const Pickers = {
             }
             container.scrollLeft = combinedWidth;
         },
+        slideRightCentered(container, elements, valueLabel) {

```

Comment:
Can't you reuse more stuff between this, `slideLeftCentered` and `centerElement`?